game.rb

```diff
@@ -0,0 +1,17 @@
+require 'date'
+require_relative 'item'
+
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(id, publish_date, multiplayer, last_played_at)
+    super(id, publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    is_expired = Time.now.year - Date.parse(@last_played_at).year > 2
+    super && is_expired
+  end
+end
```

Comment:
The game should have information on all the association classes that you described in the Item class. ie., `Label`, `Author`, and `Genre`. But in your case, the game only asks for `Author` and not the `Label` and `Genre`. As you can see in the class diagram, Item has a 1-many association with the rest of the classes and hence that should be included in the Game as it inherits from the Item. Kindly fix this issue.